certbot/docs/uninstall.rst

```diff
@@ -0,0 +1,16 @@
+=========================
+Uninstalling certbot-auto
+=========================
+
+To uninstall ``certbot-auto``, you need to do three things:
+
+1. If you added a cron job or systemd timer to automatically run
+   ``certbot-auto`` to renew your certificates, you should delete it. If you
+   did this by following our instructions, you can delete the entry added to
+   ``/etc/crontab`` by running a command like ``sudo sed -i '/certbot-auto
+   renew/d' /etc/crontab``.
```

Comment:
Do you think it would be safe to leave out the `renew` part? 

Being mindful of possibilities like: https://community.letsencrypt.org/search?q=%22certbot-auto%20-q%20renew%22

Reply:
Good idea. If the cron job contains `certbot-auto`, it should almost certainly be deleted.